en/includes/guides/branding/add-custom-layouts.md

```diff
@@ -192,10 +192,14 @@ To add a custom layout to the login page:
 
     3. Copy `<IDENTITY-APPS-HOME>/identity-apps-core/apps/authentication-portal/src/main/webapp/extensions/layouts/custom` folder (Compiled version of the layout source code) and paste it into the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint/extensions/layouts` directory.
 
-4. Refresh the browser and check out the added custom layout.
+4. To enable custom branding, go to **Branding** > **Styles & Text** > **Design** in the WSO2 Identity Server Console.
```

Comment:
is this going to asgardeo

Reply:
No, this section is not applicable or available for Asgardeo.